docs/src/documentation/05-development/04-migration-guides/v10.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: V10
+description: How to migrate to Orbit 10.0.0
+redirect_from:
+  - /migration-guides/v10/
+---
+
+# Orbit Migration Guide v10
+
+This migration guide focuses on the process of migrating from Orbit v9.1 to v10.0, as some breaking changes were introduced.
+With this guide we aim to walkthrough all the breaking changes and how they can be addressed, allowing the migration to be smoother and effortless.
+
+This version of Orbit includes the first batch of components styled with Tailwind, instead of Styled Components. Note that you can keep using Styled Components, especially on your custom components. But we recommend start migrating them to Tailwind as well.
+
+Components in Tailwind should have the same API as before, so no breaking changes are expected in that regard. However, and since some of the migrated components went through a deep refactoring, we recommend being on the lookout for unexpected and undocumented changes. Those should be reported so that we can fix them.
```

Comment:
```suggestion
Components in Tailwind should have the same API as before, so no breaking changes are expected in that regard. However, since some of the migrated components went through deep refactoring, we recommend being vigilant for unexpected and undocumented changes. Those should be reported so that we can fix them.
```